app/src/main/res/values-fr/strings.xml

```diff
@@ -45,7 +45,7 @@
   <string name="family_name">Nom de famille</string>
   <string name="add_user_example_family_name">Smith</string>
   <string name="action_search">Rechercher</string>
-  <string name="action_go_to">ID</string>
+  <string name="action_go_to">Recherche ID</string>
```

Comment:
This text was Ivan's suggestion, so I went with it.
On Oct 13, 2015 3:54 PM, "viniciusboson" <notifications@github.com> wrote:

> In app/src/main/res/values-fr/strings.xml
> <https://github.com/projectbuendia/client/pull/76#discussion_r41876539>:
>
> > @@ -45,7 +45,7 @@
> >    <string name="family_name">Nom de famille</string>
> >    <string name="add_user_example_family_name">Smith</string>
> >    <string name="action_search">Rechercher</string>
> > -  <string name="action_go_to">ID</string>
> > +  <string name="action_go_to">Recherche ID</string>
>
> Not sure, but maybe 'Rechercher par ID' would be better. It worths double
> check it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/projectbuendia/client/pull/76/files#r41876539>.
>